bindings/ruby/Cargo.toml

```diff
@@ -36,7 +36,7 @@ name = "opendal_ruby"
 magnus = { version = "0.8", features = ["bytes", "io"] }
 # 1. this crate won't be published, we always use the local version
 # 2. we use the symbolink to allow released gem to find core's source files.
-opendal = { version = ">=0", path = "./core", features = [
+opendal = { version = ">=0", path = "../../core", features = [
```

Comment:
The problem is that distributed gem can't look into parent directory for security reasons.

@tisonkun do you have a way to resolve this based on dev/distributed environment?